### PR TITLE
Support more input types

### PR DESCRIPTION
**Pull request by rooftoofwoof**
When doing a feather online query, we now support 

- [x] Struct types (by specifying inputs as Maps)
- [x] Timestamp types

### CODE REVIEW

File: src/main/java/ai/chalk/internal/arrow/FeatherProcessor.java [hunk 1]
```diff
@@ -1,160 +1,323 @@
...
-        javaToArrowType.put(Short.class, new ArrowType.Int(64, true));
-        javaToArrowType.put(Integer.class, new ArrowType.Int(64, true));
-        javaToArrowType.put(Long.class, new ArrowType.Int(64, true));
-        javaToArrowType.put(Float.class, new ArrowType.FloatingPoint(FloatingPointPrecision.SINGLE));
-        javaToArrowType.put(Double.class, new ArrowType.FloatingPoint(FloatingPointPrecision.DOUBLE));
-        javaToArrowType.put(String.class, ArrowType.LargeUtf8.INSTANCE);
-        javaToArrowType.put(Boolean.class, ArrowType.Bool.INSTANCE);
-        javaToArrowType.put(byte[].class, ArrowType.LargeBinary.INSTANCE);
+
+    public static ArrayList<StructEntry> getEntriesFromHashMap(HashMap<?, ?> obj) throws Exception {
```
Comment (AndyMoreland) [inline src/main/java/ai/chalk/internal/arrow/FeatherProcessor.java:31]: `Map`, not `HashMap`

File: src/main/java/ai/chalk/internal/arrow/FeatherProcessor.java [hunk 2]
```diff
@@ -1,160 +1,323 @@
...
+        var entries = new ArrayList<StructEntry>();
+        for (var entry : obj.entrySet()) {
+            var key = entry.getKey();
+            if (!(key instanceof String)) {
```
Comment (AndyMoreland) [inline src/main/java/ai/chalk/internal/arrow/FeatherProcessor.java:35]: Why not `Map<String, ?>` above then?

File: src/main/java/ai/chalk/internal/arrow/FeatherProcessor.java (hunk 1 above)
Comment (rooftoofwoof) [reply src/main/java/ai/chalk/internal/arrow/FeatherProcessor.java]: Done!

File: src/main/java/ai/chalk/internal/arrow/FeatherProcessor.java (hunk 2 above)
Comment (rooftoofwoof) [reply src/main/java/ai/chalk/internal/arrow/FeatherProcessor.java]: Done

**Review by AndyMoreland — APPROVED** [review]

**Pull request merged**